doc/ubuntu_release_process.md

```diff
@@ -205,12 +197,35 @@ We have daily packaging recipes that upload to the [daily ppa](https://code.laun
 
 ### When the daily recipe build fails ###
 
-The daily recipe for each release checks out master and then merges the ubuntu/$release
-and builds the package from there.  From time to time, the patches in the
-ubuntu/$release branch need to be refreshed/updated as upstream/master changes.
+The daily recipe for each release checks out tip of master, merges the ubuntu/$release
+and builds the package from there.  Daily recipe failures occur under two conditions:
+ 1. Tip of master drifts enough that patches in the ubuntu/$release branch need to be refreshed/updated/dropped as upstream/master changes.
```

Comment:
s/drifts enough/has commits which modify code that the release branch patches touch.